app/assets/stylesheets/spotlight/_nestable.scss

```diff
@@ -84,9 +84,9 @@ tr.dd-item {
 
 .dd3-content { display: block; margin: 5px 0; padding: 0 0 0 30px;
     background: #fafafa;
-    background: -webkit-linear-gradient(top, #fafafa 0%, #eee 100%);
-    background:    -moz-linear-gradient(top, #fafafa 0%, #eee 100%);
-    background:         linear-gradient(top, #fafafa 0%, #eee 100%);
+    background: -webkit-linear-gradient(to bottom, #fafafa 0%, #eee 100%);
```

Comment:
Color literals like `#fafafa` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Avoid vendor prefixes.